evm_arithmetization/src/generation/prover_input.rs

```diff
@@ -44,7 +44,9 @@ use crate::witness::util::{current_context_peek, stack_peek};
 #[derive(PartialEq, Eq, Debug, Clone, Serialize, Deserialize)]
 pub struct ProverInputFn(Vec<String>);
 
+#[allow(dead_code)]
 pub const ADDRESSES_ACCESS_LIST_LEN: usize = 2;
+#[allow(dead_code)]
 pub const STORAGE_KEYS_ACCESS_LIST_LEN: usize = 4;
```

Comment:
nit: we probably don't wanna keep all these dead code lines

Reply:
I was using those when debugging. How can I keep it for future use?

Reply:
Hmm ideally if you want to keep it then we could group all of these unused into some

```rust
#[allow(unused)]
pub(crate) debugging {

}
```

but is it really going to be useful in the future? Or was it helpful only for the transition of access list searching?

Reply:
In the end there were some tests using `ADDRESSES_ACCESS_LIST_LEN`